src/toast/toast.jsx

```diff
@@ -3,23 +3,23 @@ import PropTypes from 'prop-types'
 import classnames from 'classnames'
 import { onlyUpdateForPropTypes } from 'recompose'
 import styles from '../../src/toast/toast.styles.scss'
+import Button from './button'
 
-const onClick = (fn, handler) => () => fn(handler)
-
-const Toast = ({ style, className, title, button, print, position, type, remove }) => {
+const Toast = ({ style, className, title, button, print, position, type }) => {
   const classes = classnames(
     styles.toast,
     className,
     styles[position],
     styles[type],
-    { [styles.print]: print })
+    { [styles.print]: print },
+  )
 
   return (
     <div style={style} className={classes}>
       <span className={styles.text}>{title}</span>
-      {button && button.text && <button onClick={onClick(remove, button.handler)}>
-        {button.text}
-      </button>}
+      {button && button.text && (
```

Comment:
This component should not have to know that button has a `text` field.
I would prefer to see this here:
```
{button && (
   <Button {...button} type={type} />
)}
```

And then into button component : 
```es6
const Button = ({ text, onClick, /* ... */ }) => {
  if (!text) return null

  return (
   /* ... */
  ) 
}
```